EnglishTranslation-master/QUEST_UNUSED.tsv

```diff
@@ -911,6 +913,7 @@ QUEST_UNUSED_20151022_000912	We can't just rely on the goddess.{nl}We should do
 QUEST_UNUSED_20151022_000913	We, Kedoran Merchant Alliance helped the goddesses secretly until now.{nl}But, since the world turned into a chaos, we are going to help her actively.
 QUEST_UNUSED_20151022_000914	We are in a trouble..{nl}There are so many things to retrieve, but I don't know what people are doing.
 QUEST_UNUSED_20151022_000915	Ah, so Leopoldas sent you here?{nl}Finally, it seems he has stopped being so stubborn.{nl}
+QUEST_UNUSED_20151022_000916	A long time ago, I was sending the sacred objects to Fedimian.{nl}But, I didn't realise it was the day of Medzio Diena.{nl}
```

Comment:
`it was the day of Medzio Diena` -> `it was on Medzio Diena`

Medzio Diena = Day of the Divine Tree